docs/scripting/scripts.lib/ok.mdx

```diff
@@ -19,13 +19,15 @@ No known parameters.
 ### Return
 
 ```js
-Success
+Success;
 ```
 
 Same as:
 
 ```js
-{ok:true}
+{
+  ok: true;
+}
 ```
 
 ## Example
```

Comment:
This is a complicated one-- should we be adding semicolons to the example below?

though-- I'm assuming that they are omitted and 'var' is being used instead of const for char count reasons.